includes/core-changes/globalization/5.0/icu-globalization-api.md

```diff
@@ -0,0 +1,46 @@
+### Globalization APIs use ICU libraries on Windows
+
+.NET 5.0 and later versions use [International Components for Unicode (ICU)](http://site.icu-project.org/home) libraries for globalization functionality when running on Windows 10 May 2019 Update or later.
+
+#### Change description
+
+Previously, .NET libraries used [National Language Support (NLS)](/windows/win32/intl/national-language-support) APIs for globalization functionality. For example, NLS functions were used to set the user's locale, sort strings, and format dates and times in the appropriate culture.
```

Comment:
> NLS functions were used to set the user's locale, sort strings, and format dates and times in the appropriate culture.

I am not sure if this is clear. it is better to rephrase it to something like:
`NLS functions were used to get culture data (e.g. date and time format patterns, currency symbols,...etc.), compare strings and do string casing in the appropriate culture`